Solutions/Menes.PetStore.Specs/Steps/Steps.cs

```diff
@@ -19,15 +19,15 @@ namespace Menes.PetStore.Specs.Steps
     [Binding]
     public class Steps
     {
-        private static readonly HttpClient HttpClient = HttpClientFactory.Create();
-
         private static readonly Uri BaseUri = new("http://localhost:7071");
 
         private readonly ScenarioContext scenarioContext;
+        private readonly HttpClient httpClient;
 
         public Steps(ScenarioContext scenarioContext)
         {
             this.scenarioContext = scenarioContext;
+            this.httpClient = new HttpClient();
```

Comment:
Is it not better to use `HttpClientFactory.Create()` rather than `new HttpClient()`?